test/functional/commands/find/by-uiautomator-e2e-specs.js

```diff
@@ -67,7 +67,9 @@ describe('Find - uiautomator', function () {
     let el = await driver.findElOrEls('-android uiautomator', 'new UiSelector().clickable(true).className(android.widget.TextView).index(1)', false);
     await driver.getText(el.ELEMENT).should.eventually.equal('Accessibility');
   });
-  it('should find an element with recursive UiSelectors', async () => {
+  it('should find an element with recursive UiSelectors', async function () {
+    // TODO: figure out why this fails with 7.1.1
+    if (await driver.adb.getApiLevel() >= '24') this.skip();
```

Comment:
from mocha docs: 

> Best practice: To avoid confusion, do not execute further instructions in a test or hook after calling this.skip().

I looked this up because I was indeed confused whether `skip()` altered control flow or not. Looks like it does! Maybe a `return this.skip()` just to be very clear?